tests/TestCase/Shell/Task/CommandTaskTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations\Test\TestCase\Shell\Task;
+
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOutput;
+use Cake\TestSuite\TestCase;
+
+/**
+ * Class TestCompletionStringOutput
+ *
+ */
+class TestCompletionStringOutput extends ConsoleOutput
+{
+
+    public $output = '';
+
+    protected function _write($message)
+    {
+        $this->output .= $message;
+    }
+}
+
+/**
+ * Class CommandTaskTest
+ */
+class CommandTaskTest extends TestCase
+{
+
+    /**
+     * Instance of ConsoleOutput
+     *
+     * @var \Cake\Console\ConsoleOutput
+     */
+    public $out;
+
+    /**
+     * setUp method
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->out = new TestCompletionStringOutput();
```

Comment:
Yeah, I don't know why I haven't thought of that.
Just pushed a new commit with the change.
Thanks.